public/index.html

```diff
@@ -27,6 +27,12 @@
   </head>
   <body>
     <div id="root"></div>
+
+    <a href="./#/CustomQuiz">Click Here to create a CUSTOM QUIZ!</a> <br/><br/>
```

Comment:
Cal, react routing takes care of this. It's best practice to keep the index.html is clean and have our index.js handle routing.